### PR TITLE
Add cmdlet Get-AzAccessToken

### PR DESCRIPTION
**Pull request by erich-wang**
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Accounts/Accounts/help/Get-AzAccessToken.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
+
+## INPUTS
+
+### None
+
+## OUTPUTS
+
+### System.String
```
Comment (erich-wang) [inline src/Accounts/Accounts/help/Get-AzAccessToken.md:95]: In addition to raw access token, more info should returned: ExpiresOn, TenantId, SubscriptionId

File: src/Accounts/Accounts/help/Get-AzAccessToken.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+# Get-AzAccessToken
+
+## SYNOPSIS
+Get raw access token
+
+## SYNTAX
+
+### KnownResourceTypeName
+```
+Get-AzAccessToken -ResourceTypeName <String> [-TenantId <String>] [-DefaultProfile <IAzureContextContainer>]
```
Comment (dingmeng-xue) [inline src/Accounts/Accounts/help/Get-AzAccessToken.md:17]: What's the difference between ResourceType and RP? I'm confused about this name

Comment (erich-wang) [reply src/Accounts/Accounts/help/Get-AzAccessToken.md]: ResourceTypeName should be same as RP name if possible, but there's no RP name for some services, e.g. Graph, ResourceManager, for those I'll use common used names.

File: src/Accounts/Accounts/help/Get-AzAccessToken.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ResourceTypeName
+Optional resouce type name, supported values: AadGraph, Analysis, Arm, Attest, DataLake, KeyVault, OperationInsights, Synapse. Default value is Arm if not specified.
```
[the remark this answers is not in the document]
Comment (erich-wang) [reply src/Accounts/Accounts/help/Get-AzAccessToken.md]: Yes, but it should not a matter as they're provided by ArgumentCompleter

**Review by isra-fel — DISMISSED** [review]
LGTM

**Review by isra-fel — APPROVED** [review]

**Pull request merged**